Dalamud/Interface/Utility/ImGuiHelpers.cs

```diff
@@ -533,6 +534,12 @@ public static ushort[] CreateImGuiRangesFrom(params UnicodeRange[] ranges)
     public static unsafe ImFontPtr OrElse(this ImFontPtr self, ImFontPtr other) =>
         self.NativePtr is null ? other : self;
 
+    /// <summary>Converts the wrong struct to a correct struct.</summary>
+    /// <param name="glyph">The glyph being corrected.</param>
+    /// <returns>The correct struct.</returns>
+    internal static unsafe ref ImFontGlyphReal ToReal(in this ImFontGlyphPtr glyph) =>
```

Comment:
Did you leave this in by accident? I can't see it being used anywhere here.

Reply:
Yeah, that was written while testing stuff and eventually it became unused.